cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -193,6 +196,9 @@ func (s *CMServer) Run(_ []string) error {
 	controllerManager := replicationControllerPkg.NewReplicationManager(kubeClient, replicationControllerPkg.BurstReplicas)
 	go controllerManager.Run(s.ConcurrentRCSyncs, util.NeverStop)
 
+	daemonManager := daemon.NewDaemonManager(kubeClient)
```

Comment:
No action required? They are already using the CMServer. There is not "enable" flag...